macro-test/src/test/scala/scala/pickling/generator/scalasymbols/TestScalaSymbols.scala

```diff
@@ -54,34 +54,34 @@ class TestScalaSymbols extends FunSuite {
 
   test("fieldTypesWithTypeArgs") {
     assert(Compat.varTypes[ParentTag.Foo[String, Int]].toSet == Set(
-      "scala.Tuple2[java.lang.String,scala.Int]", "scala.Int"
+      "(String, Int)", "Int"
```

Comment:
This is literally just a test.  The macro is ONLY used to test that symbols are pulled out.

Not meant to be human friendly, so no, it's not meant to be taken seriously.  It's only so we can test if things inside the macro are working.